agent/app/agent_capability_windows_test.go

```diff
@@ -260,3 +260,47 @@ func TestAppendGMSACapabilitiesFalse(t *testing.T) {
 
 	assert.Equal(t, len(expectedCapabilities), len(capabilities))
 }
+
+func TestAppendFSxWindowsFileServerCapabilities(t *testing.T) {
+	var inputCapabilities []*ecs.Attribute
+	var expectedCapabilities []*ecs.Attribute
+
+	expectedCapabilities = append(expectedCapabilities,
+		[]*ecs.Attribute{
+			{
+				Name: aws.String(attributePrefix + capabilityFSxWindowsFileServer),
+			},
+		}...)
+
+	agent := &ecsAgent{
+		cfg: &config.Config{
+			FSxWindowsFileServerCapable: true,
+		},
+	}
+
+	capabilities := agent.appendFSxWindowsFileServerCapabilities(inputCapabilities)
+
+	assert.Equal(t, len(expectedCapabilities), len(capabilities))
+	for i, expected := range expectedCapabilities {
+		assert.Equal(t, aws.StringValue(expected.Name), aws.StringValue(capabilities[i].Name))
+		assert.Equal(t, aws.StringValue(expected.Value), aws.StringValue(capabilities[i].Value))
+	}
+}
+
+func TestAppendFSxWindowsFileServerCapabilitiesFalse(t *testing.T) {
```

Comment:
These tests are good candidates for table driven tests. 
Refer https://github.com/golang/go/wiki/TableDrivenTests

Reply:
this test is identical to ```TestAppendGMSACapabilitiesFalse```